qtconsole/console_widget.py

```diff
@@ -396,6 +399,7 @@ def eventFilter(self, obj, event):
             text widgets.
         """
         etype = event.type()
+        self._trigger_is_complete_callback()
```

Comment:
I haven't tested with a kernel that doesn't support `is_complete`, but I just killed the reply code to simulate no response, and it acts as expected: it always considers the command incomplete, and continues with no indent.